src/main/java/frc/robot/DTButton.java

```diff
@@ -1,39 +1,40 @@
 package frc.robot;
 
 import edu.wpi.first.wpilibj.Joystick;
-//Mike, this one's for you to handle
+// Mike, this one's for you to handle
 
 public class DTButton {
+    private final Joystick joystick;
+    private final int      buttonID;
 
-    private boolean old, current;
-    private Joystick stick;
-    private int button;
+    private boolean old;
+    private boolean current;
 
-    public DTButton(Joystick stick, int button) {
-        this.stick = stick;
-        this.button = button;
+    public DTButton(Joystick joystick, int buttonID) {
+        this.joystick = joystick;
+        this.buttonID = buttonID;
```

Comment:
We haven't standardized on the use of `this` for local fields. Most of our code doesn't use it, so I would suggest going the other way (remove all uses of `this` except where required, e.g. to pass a reference to this into a method of another class).

Reply:
All instances removed

Reply:
(Except the two you highlighted there because changing them would require variable name changes that reduce clarity)